src/plugins/index_pattern_management/public/components/experimental_callout/experimental_callout.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+
+export const ExperimentalCallout = () => {
+  return (
+    <>
+      <EuiCallOut
+        title={TITLE}
+        iconType="alert"
+        color="warning"
+        data-test-subj="index-pattern-experimental-callout"
+      >
+        <p>
+          <EuiText data-test-subj="index-pattern-experimental-callout-text">
+            {DESCRIPTION_FIRST_PART}
+            <b>{DATASOURCE_CONNECTION}</b>
+            {DESCRIPTION_SECOND_PART}
+            <b>{DEFAULT}</b>
+            {DESCRIPTION_THIRD_PART}
+          </EuiText>
+        </p>
+      </EuiCallOut>
+      <EuiSpacer size="m" />
+    </>
+  );
+};
+
+const TITLE = i18n.translate('indexPatternManagement.experimentalFeatureCallout.title', {
```

Comment:
I havent had the chance to use the localization features yet but calling them like this would only run localization once. If we switch our locale after the initial page load, these string will not change.

Why dont we use the `FormattedMessage` component from `@osd/i18n/react` instead of declaring each string separately like this?

Reply:
This will be changed in next PR

Reply:
#2543